<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuNode</class>
 <widget class="QWidget" name="MenuNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>31</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>31</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>31</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MenuNode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MenuNode {
	background-color: transparent;
}

QWidget#leftContainer {
	background-color: transparent;
}

QWidget#centerContainer {
	background-color: transparent;
	background-image: url('./data/ui/images/menus/icon_bground_center.png');
	background-position: top left;
	background-repeat: repeat-x;
}

QWidget#textWidget {
	background-color: transparent;
}

QWidget#iconWidget {
	background-color: transparent;
}

QWidget#rightContainer {
	background-color: transparent;
	background-image: url('./data/ui/images/menus/editbtns_bground_end');
	background-position: top left;
	background-repeat: no-repeat;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="leftContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="iconWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="centerContainer" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="textWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="rightContainer" native="true">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
